src/lib/math.js

```diff
@@ -943,6 +992,12 @@ const $builtinmodule = function (name) {
             $textsig: "($module, x, /)",
             $doc: "Return True if x is a NaN (not a number), and False otherwise.",
         },
+        lcm: {
```

Comment:
Bonus - python 3.9 also added arbirtray arguments to the `gcd` function.
Feel free to make that outside the scope of this PR. But it might be a nice feature to add.

Reply:
After finishing with this pr I plan to do other missing math methods. If I will have spare time I will try to get to updating gcd)